Automattic/wp-calypso
# Mock: Light importer, import colors flow

#### Changes proposed in this Pull Request

* Created playground to mimic the whole light importer flow.

#### Testing instructions

* Go to: `/start/import-light/static?siteSlug={SITE_SLUG}`
* Enter any website (it is required to be a valid address)
* Check if the final step is site editor

Related to #
